additive_animations/src/main/java/at/wirecube/additiveanimations/additive_animator/AdditiveAnimationStateManager.java

```diff
@@ -161,10 +155,21 @@ void cancelAnimation(String propertyName) {
                 cancelledAppliers.add(applier);
             }
         }
+        removeStateManagerIfAccumulatorSetIsEmpty();
```

Comment:
executing the function before actually calling 
`mAdditiveAnimationAccumulators.removeAll(cancelledAppliers)`
doesn't seem correct 🤔

Reply:
Hah, good catch. Fixed